### ISSUE TITLE
Tell users of the feature/redrawhook branch to switch back to master

### ISSUE THREAD

**Issue by danielshahaf** [zsh-users/zsh-syntax-highlighting#655]
(Context: we've received multiple support questions which were resolved by having the user switch to the feature/redrawhook branch, e.g., #654.)

After merging feature/redrawhook, arrange for users of that branch to see a message "You may switch back to master".

**Comment by danielshahaf** [comment]
Proposed patch:

```diff
diff --git a/zsh-syntax-highlighting.zsh b/zsh-syntax-highlighting.zsh
index 3608203..f01da13 100644
--- a/zsh-syntax-highlighting.zsh
+++ b/zsh-syntax-highlighting.zsh
@@ -589,6 +589,20 @@ if (( $+X_ZSH_HIGHLIGHT_DIRS_BLACKLIST )); then
   unset X_ZSH_HIGHLIGHT_DIRS_BLACKLIST
 fi
 
+# Tell users of this branch (feature/redrawhook) to switch back to master.
+#
+# We don't want to nag them on every new shell, because that would be annoying.
+# We can't nag them once and never again, because we can't keep state.
+# Therefore, compromise: warn infrequently.
+#
+# Magic number: 63 is one less than a power of two.
+if (( (RANDOM & 63) == 42 )); then
+  print -r -- $'zsh-syntax-highlighting: You are using the feature/redrawhook branch.'
+  print -r -- $'zsh-syntax-highlighting: That branch has been merged to master and will not receive any further commits.'
+  print -r --  "zsh-syntax-highlighting: Consider switching ${(qq)0:h} back to the 'master' branch."
+fi
+
+
 # Restore the aliases we unned
 eval "$zsh_highlight__aliases"
 builtin unset zsh_highlight__aliases
```

This will print the message in 1.6% of new shells.

If anyone is aware of any `setopt APL_JUNKIE_BITOPS`-style knobs that make the above unportable, please say so ☺

**Comment by danielshahaf** [comment]
Fixed in ffed6e165475.
